src/version.rs

```diff
@@ -461,6 +461,8 @@ pub fn authors(req: &mut Request) -> CargoResult<Response> {
 }
 
 /// Handles the `DELETE /crates/:crate_id/:version/yank` route.
+/// This does not delete a crate; it makes a crate accessible
+/// only to Cargo.lock files with the yanked version.
```

Comment:
I would say "it makes a crate accessible only to crates that already have a Cargo.lock containing this version." We could also elaborate a bit more on why deleting isn't implemented (deleting a crate could break people's builds) and that yanking is meant to prevent crates from *starting* to depend on a crate.